batch/batch/front_end/templates/table_search.html

```diff
@@ -64,4 +64,13 @@
       </form>
     </div>
   </div>
+  <script>
+    document.getElementById("{{ component_name }}-input-box")
+      .addEventListener("keydown", function(e) {
+        if (e.which === 13 && e.metaKey) {
```

Comment:
I think windows users will find this surprising because meta either doesn't exist or is the windows key. Can we do `(e.metaKey || e.ctrlKey)`?